### ISSUE TITLE
[Feedback] The icon in the system tray is not very prominent.

### ISSUE THREAD

**Issue by nesteiner** [localsend/localsend#1395]
![image](https://github.com/localsend/localsend/assets/46296608/e5170071-1697-432e-b64a-0152411e45c7)

as you can see, the icon is not very clear, and I don't know how to set it.
by the way I am using kde plasma

**Comment by Tienisto** [comment]
On Linux, the icon color is always white. See #649 
I don't know if it is possible to detect the task bar background color to determine the correct icon color

**Comment by sergd88** [comment]
> On Linux, the icon color is always white. See #649 I don't know if it is possible to detect the task bar background color to determine the correct icon color

In my opinion, it's a very, very bad idea to use only a white icon because besides GNOME (which indeed defaults to a dark panel), there are other desktop environments, like KDE in this case, which natively support both light and dark themes, and it's also a very popular Linux desktop environment. Support for light theme (including panel) exists in many other popular DE as well. Therefore, I consider such a decision very poor because in the light theme, it really looks like a bug. If monochrome icon variants are already being used, which indeed look somewhat better, at least in my opinion, then in that case, it's absolutely necessary to use both light and dark variants of the icons. But then, additional implementation of automatic or eventually manual switching (which few people still do) based on the system theme will be required. However, I suspect automatic switching could be quite challenging and laborious as the implementation may vary depending on the desktop environment, so using a colored icon is the simplest option and not the worst one. Google Drive still doesn't support monochrome icons either 😏

Previously, on Linux, I really disliked how the Telegram icon looked in the menu bar because it looked something like this:

![teleg](https://github.com/localsend/localsend/assets/125032334/5e724958-127d-44bc-9b98-03f9cfa88a27)


So, it also displayed a message count indicator, and the icon was practically invisible underneath it. That's why I used [Hardcode-Tray](https://github.com/bilelmoussaoui/Hardcode-Tray) for this purpose, which is designed to replace colored icons with monochrome ones. And with it, the icon looked more or less the same as it's implemented now in Telegram.

Perhaps those who are not satisfied with the colored icon version should still consider similar projects. Because the colored icon version doesn't create as much inconvenience as using a purely white icon.

So, to summarize, it's still worth using a colored icon until automatic switching of the monochrome icon based on the system theme is implemented.

**Comment by Septicity** [comment]
Setting either color to be the default would be a bad idea... aren't symbolic icons intended for this exact issue? The color icon could be the default, while a symbolic icon (a simple SVG colored by the OS) would be available for panels/taskbars/system trays that are customized for it...

**Comment by sudoshindo** [comment]
Yes, symbolic icons please.
A lot of other applications in my tray already use them.

**Comment by teito-dev** [comment]
It's not just a white icon issue, it's appearing transparent on awesomewm tiling window manager linux's tray too

![Image](https://github.com/user-attachments/assets/24bdc6eb-706c-4c7b-8307-16ac25e034d0)

**Comment by TheFighter137** [comment]
My own situation on Linux Mint is similar to OP's when using light mode :

<img width="714" height="224" alt="Image" src="https://github.com/user-attachments/assets/4bbc5aa4-c071-4485-91f1-86586e244903" />
